app/models/form/setup/pages/location.rb

```diff
@@ -0,0 +1,20 @@
+class Form::Setup::Pages::Location < ::Form::Page
+  def initialize(_id, hsh, subsection)
+    super("location", hsh, subsection)
+    @header = ""
+    @description = ""
+    @questions = questions
+    @depends_on = [{
+      "supported_housing_schemes_enabled?" => true,
+      "needstype" => 2,
+      "scheme_has_multiple_locations?" => true,
+    }]
+    @derived = true
```

Comment:
Not sure if it makes sense for the "page" to be derived

Reply:
It doesn't this should be on the `question`